airbyte-ci/connectors/connector_ops/tests/test_utils.py

```diff
@@ -199,5 +199,7 @@ def test_get_all_gradle_dependencies(with_test_dependencies):
 def test_get_all_connectors_in_repo():
     all_connectors = utils.get_all_connectors_in_repo()
     assert len(all_connectors) > 0
-    assert all([isinstance(connector, utils.Connector) for connector in all_connectors])
-    assert all([connector.metadata is not None for connector in all_connectors])
+    for connector in all_connectors:
+        assert isinstance(connector, utils.Connector)
+        assert connector.metadata is not None
+        assert connector.documentation_file_path.exists()
```

Comment:
This test would fail if a connector does not have a doc. Do we want this lib test to fail in this scenario? At the moment missing doc would lead to a failing QA check. What do you think about asserting the documention path structure instead of checking its existence?

Reply:
Good call. I updated to check if a connector has an airbyte.com documentation url before running this check
https://github.com/airbytehq/airbyte/pull/31324